src/routes/users.js

```diff
@@ -12,19 +14,14 @@ const getUser = async (req, res) => {
 }
 const updateUser = () => {}
 const deleteUser = () => {}
-const checkId = (req, res, next , val ) => {
+const checkId = (req, res, next, val) => {
```

Comment:
@beblicarl could you shed some light on what this should do?

Reply:
the user.js file serves as the template of how the post.js should be built. I worked on the post.js file @jocrah

Reply:
Got it but our pr here has changes in the user.js so trying to confirm some things there as well. just curious to know what the `checkId` does or you mean to say it wasn't added by you?

Reply:
It was initially meant to be used to check for authentication , but it is something I will take out in my next commit @jocrah

Reply:
Okay, I think since we're learning we should go by the ideal approach.

Let's always make sure our changes mainly deal with just what we are mainly working on or the scope of the issue at hand.

Otherwise, it becomes something the reviewer has to review and that becomes a bit confusing as well

So instead of doing additional changes, you can instead reset the file to remove your changes by doing `git checkout origin/main <file_path>` and then commit it.

Kindly let me know if you get me.

Reply:
yeah, i get you. You are right, it doesn't have anything to do with the actual issue at hand. I will rest the file to remove my changes

Reply:
@jocrah  i have reset the user.js now